components/index.md

```diff
@@ -43,36 +43,51 @@ All changes are backward compatible.
+and others. 
 
 Experimental components may not progress to the Beta and GA stages.
 
-Nevertheless, these components can address obscure use cases and deliver unique integrations. As their code is public, they can serve as a foundation for custom forks, 
-which could also include Generic Extractor configurations.
+Nevertheless, these components can address obscure use cases and deliver unique integrations. Their code is public so they can serve as a foundation for custom forks, 
+including Generic Extractor configurations.
 
-Many of these components also serve for our internal purposes, and we decided to share them publicly for the benefit of our community.
+Many of these components also serve our internal purposes, and we decided to share them publicly to benefit our community.
 
 ### Private / Unlisted
-Some components are still unlisted for various reasons. The full list of components available in each stack is accessible via the public [Storage API index call](https://keboola.docs.apiary.io/#reference/miscellaneous/api-index/component-list). 
+Some components are still unlisted for various reasons. Each stack's full list of components is accessible via the public [Storage API index call](https://keboola.docs.apiary.io/#reference/miscellaneous/api-index/component-list). 
 Many of these are 3rd-party components. Users can add these components via their ID, but we cannot guarantee their functionality. 
 
-We may share our pre-release versions that exist in private Beta with our test user groups. In such case, you will receive a component ID, using which you can create the configuration. 
+We may share our pre-release versions that exist in private Beta with our test user groups. In such a case, you will receive a component ID, which you can use to create the configuration. 
 These components will eventually transition to a public Beta.
 
+## Configuration Description
+After configuring the component, it is a good practice to describe what it does.
+
+{: .image-popup}
+![Screenshot - Add Description button](/components/description-button.png)
+
+Click the **Add description** button to write the description manually. You can also click the **Generate description** button to have our AI service write a suggestion for you that you can modify. 
+Note that the configuration will be sent to the Microsoft Azure OpenAI service when using the [AI service](/overview#ai-assistance). Even though the contents are not used to train the model, as a best practice, 
+you should not store unencrypted secrets such as API keys and passwords in the configuration (for example, in Python transformation code). Variables and encrypted configuration parameters are automatically 
+obfuscated.
+
+{: .image-popup}
+![Screenshot - AI generated description](/components/description-button.png)

```

Comment:
Should be components/description-generated.png

Reply:
Fixed in 660de11f36f22a0b11f7e024fd86b066eabf3fcb

Reply:
https://help.keboola.com/components/#configuration-description